src/SqlServerCache/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Data;
+using System.Data.SqlClient;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace SqlServerCache
```

Comment:
Should discuss with engineering team. @kichalla can you start a thread and also include @Tratcher ? I do think we should use long names. These things aren't typed almost ever.